content/Dev/python_good_practices_2020.rst

```diff
@@ -0,0 +1,371 @@
+Python good practices in early 2020
+###################################
+
+:tags: python
+:date: 2020-04-01
+
+
+I had the chance to spin off a new Python project recently, and that was a good opportunity to look at recent trends :) This article goes through some of the choices we made, knowing that almost everything is obviously debatable.
+
+
+Environment
+-----------
+
+Since we publish the app as a Docker container, we don't have to support multiple Python environments (eg. with `tox <https://tox.readthedocs.io>`_). Even if contributors may want to use `Pyenv <https://github.com/pyenv/pyenv>`_ to overcome the limitations of their operating system.  
```

Comment:
`Even` is a conjunction -- it should either be part of the previous sentence or we should reframe the sentence (something like `On the other hand, contributors...`)